src/main.js

```diff
@@ -1,4 +1,5 @@
 import { createApp } from 'vue'
 import App from './App.vue'
+import router from './router'
 
-createApp(App).mount('#app')
+createApp(App).use(router).mount('#app')
```

Comment:
I would restructure this to be cleaner because it will get messy as we add more things to it. So follow this pattern:
```const app = createApp(App)
app.use(VueAxios, axios)
app.use(router)
app.use(store)
app.component('download-csv', JsonCSV)
app.mount('#app')```